user/pgp2ssh/template.py

```diff
@@ -0,0 +1,16 @@
+pkgname = "pgp2ssh"
+pkgver = "0.1"
+pkgrel = 0
+archs = ["x86_64"]
+build_style = "go"
+hostmakedepends = ["go"]
+pkgdesc = "Convert PGP/GPG private keys to SSH private keys"
+maintainer = "as400 <as400@smail.net.pl>"
+license = "MIT"
+url = "https://github.com/pinpox/pgp2ssh"
+source = f"{url}/archive/refs/heads/main.tar.gz"
```

Comment:
Please don't use "main", because the repository can change at any time. If you're using Git as the source, then use a specific commit.